buildkite/scripts/run-test-executive.sh

```diff
@@ -5,6 +5,11 @@ TEST_NAME="$1"
 MINA_IMAGE="gcr.io/o1labs-192920/mina-daemon:$MINA_DOCKER_TAG-devnet"
 ARCHIVE_IMAGE="gcr.io/o1labs-192920/mina-archive:$MINA_DOCKER_TAG"
 
+if [[ "${TEST_NAME:0:4}" == "opt-" ]] && [[ "$RUN_OPT_TESTS" == "" ]]; then
```

Comment:
what is this block of code for?  is this your way of temporarily removing your test from CI?  seems like this code should be removed now

Reply:
This block doesn't run `opt-XXX` integration tests unless `RUN_OPT_TESTS` is provided.

Test from this PR fails with around 25% probability (i.e. isn't reliable, but is informative when launched many times). Also it's expensive (30 instances and 1.5 hour), hence it's switched off by default but can be laucnhed manually through Buildkite